fixtures/README.md

```diff
@@ -0,0 +1,48 @@
+# JSON fixture format
+
+Note that this fixture format is not intended to be used directly, as it has no way of versioning and may change without notice.
+
+The [JS Schema](https://github.com/molnarg/js-schema) can be found in the [schema.js](schema.js) file. See there for the details of allowed properties or values. The purpose of this document is to give a high-level overview of the concepts used.
+
+
+## Goals
+
+The JSON fixture format intends to be
+
+* readable by both humans and machines
+* easily extendable
+* as general as possible to include information for many fixture formats
+* abstract where possible, specific where needed
+
+
+## Directory structure
+
+The manufacturer of a fixture is determined by its toplevel directory (relative to the `fixtures` directory), the fixture key is the filename without extension. Manufacturer data is stored in [manufacturers.json](manufacturers.json).
+
+
+## Modes
+
+A fixture can have multiple *modes* (also sometimes called *personalities*) like "Basic 3-channel mode" or "Extended 5-channel mode". Our modes are not allowed to have the word "mode" in them, as it is automatically appended at the end.
+
+A mode can contain the `physical` property to override specific physical data of the fixture. E.g. one mode could set the `panMax` value different than the fixture default.
```

Comment:
If you talk about overriding physical data, it might be a good idea introducing the fixture physical before.